fastlane/.env.SAMPLE

```diff
@@ -9,3 +9,6 @@ GITHUB_PULL_REQUEST_API_TOKEN=
 GITHUB_PULL_REQUEST_ASSIGNEES=
 GITHUB_PULL_REQUEST_REVIEWERS=
 GITHUB_PULL_REQUEST_TEAM_REVIEWERS=
+
+# Optional for updating changelog. Can be the same as GITHUB_PULL_REQUEST_API_TOKEN and improves issues with github api rate limits.
+GITHUB_TOKEN=
```

Comment:
Adding this token avoided the rate limit issues I had when testing this, so I added it here, since it can be useful.